### DISCUSSION TITLE
[SOLVED] [Font] Can we set fonts for Latin Text and Complex Script separately as in MS-Word?

### DISCUSSION THREAD

**Discussion by Starboy-Xo** [martinrotter/rssguard#660] [Q&A]
This picture is taken from MS-Word Font Setting:

![image](https://user-images.githubusercontent.com/15247707/158656416-c4bdccf5-a962-48d3-9fe5-3ba766bf862b.png)

As you can see in the picture, **Word** has ability to determine and set fonts to the **Latin Text** and **Complex Script** i.e. Arabic, Urdu etc.  Separately.

So, is it possible that we have this ability in RSSGuard?

If yes, let me know, I will convert it into Feature Request.

**Comment by martinrotter** [comment]
Hello.

Not technically possible, the only way I see is specific for Linux, where you can fine tune Fontconfig.

https://wiki.archlinux.org/title/Font_configuration/Examples#Default_fonts

**Reply by Starboy-Xo** [reply]
Hmm, alright, thanks.
